travis/update-release.py

```diff
@@ -59,8 +59,44 @@ def count_severity(filename):
     return []
 
 
+def get_container_images_data(yaml_data):
+    # Check if yaml_data is not None before accessing its keys
+    #names = [ "acc-provision-operator","aci-containers-host","aci-containers-controller","cnideploy","aci-containers-operator","openvswitch","opflex"]
+    RELEASE_TAG = os.environ.get("RELEASE_TAG")
+    for release_idx, release in enumerate(yaml_data["releases"]):
+        rName = release.get("release_name")
+        if rName == RELEASE_TAG[:8]+"z":
```

Comment:
are you sure it will be always 7 chars in length? And I think it should be ".z"

Reply:
meaning if we are sticking to 8 chars. If we increase it . It might get into error. Why not use split and join using "." . Because if someone tries some different tag and not 8 char dots it might error put. Just a trivial comment. Think more I meant to say